pkg/resource/notebook_instance_lifecycle_config/testdata/v1alpha1/readone/observed/creating_after_describe.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstanceLifecycleConfig
+metadata:
+  creationTimestamp: null
+  name: unit-testing-notebook-instance-lifecycle-config
+spec:
+  notebookInstanceLifecycleConfigName: unit-testing-notebook-instance-lifecycle-config
+  onCreate:
+  - content: ZWNobyAiRW50ZXJpbmcgb25DcmVhdGUi
+  onStart:
+  - content: ZWNobyAiRW50ZXJpbmcgb25TdGFydCI=
+status:
+  ackResourceMetadata:
+    arn: arn:aws:sagemaker:us-west-2:123456789012:notebook-instance-lifecycle-config/unit-testing-notebook-instance-lifecycle-config
+    ownerAccountID: ""
+  conditions:
+  - status: "True"
+    type: ACK.ResourceSynced
+  creationTime: "0001-01-01T00:00:00Z"
+  lastModifiedTime: "0001-01-01T00:00:00Z"
```

Comment:
does the lastmodifiedtime show up even on the initial creation ? (just wondering in the context of applicationautoscaling)

Reply:
yes I believe so

Reply:
thanks!